### DISCUSSION TITLE
System bootloop

### DISCUSSION THREAD

**Discussion by Lionheartzz** [Mr-MIBonk/M.I.B._More-Incredible-Bash#257] [Q&A]
Sorry to disturb.
I had tried install M.I.B on a Aisin unit, it seems all installation completed successfully.
However, I felt there is something going wrong, and the system start to run slower, reboot and loop in some minutes.
As I checked my log, it is surprised that are short and totally different from others.
Attached log file for reference, could any master point out the faulty and give a direction of way out. Many thanks.
[MU1002-AW7-MON21.12.1800060070.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11783346/MU1002-AW7-MON21.12.1800060070.log)

**Comment by harman-f** [comment]
Which train does your unit run?

yes, log is strange. Get a different SD card, this looks like something is wrong here. 

In Addition get latest version directly from repo. It looks like you have a quite old train which does not contain certain binaries M.I.B 3.6 is using. Newest version is doing stuff a different way, which also will work on your unit.

**Reply by Lionheartzz** [reply]
Thanks for you advise, my currently train is MHI2Q_CN_AUG22_K1004. I had replaced the SD card, and try in M.I.B 3.4.1, The system seems running well, and log file was written fine.
[MU1003-AW7-MON21.12.1800060070.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11795619/MU1003-AW7-MON21.12.1800060070.log)
But the MMI still reboot and loop, what I only can do is letting the M.I.B autorun, and stopped in firmware update complete page, connect telnet with ethernet, manually start M.I.B by typing /net/mmx/fs/sda0/start. I had tried fixed EL error, manually flash rcc fs0, and almost all options in the menu, unfortunately the MMI is still reboot in loop.
Do you have any idea where I goes wrong from the log file, Thank you!
